libflux/src/flux/semantic/builtins.rs

```diff
@@ -269,6 +269,19 @@ pub fn builtins() -> Builtins<'static> {
                 "splitRegexp" => Node::Builtin("forall [] (r: regexp, v: string, i: int) -> [string]"),
                 "getString" => Node::Builtin("forall [] (r: regexp) -> string"),
             }),
+            "runtime" => Node::Package(maplit::hashmap! {
+                "version" => Node::Builtin("forall [] () -> string"),
+            }),
+            "slack" => Node::Package(maplit::hashmap! {
+                "validateColorString" => Node::Builtin("forall [] (color: string) -> string"),
+            }),
+            "socket" => Node::Package(maplit::hashmap! {
+                "from" => Node::Builtin("forall [t0] (url: string, ?decoder: string) -> [t0]"),
+            }),
+            "sql" => Node::Package(maplit::hashmap! {
+                "from" => Node::Builtin("forall [t0] (driverName: string, dataSourceName: string, query: string) -> [t0]"),
+                "to" => Node::Builtin("forall [t0, t1] (<-tables: [t0], driverName: string, dataSourceName: string, table: string, ?batchSize: int) -> [t1] "),
```

Comment:
I think it's safe to make the `<-tables` argument and the return type to the same type variable here.  What you have here is correct, but if we make `<-tables` accept a `[t0]` and the return type also be `[t0]`, then the result of type inference will be better.

I'll step through how I came to that conclusion: I looked at the `Process()` method for the `sql.to`, which is here: https://github.com/influxdata/flux/blob/cb41c28d3391d1fc0285e3ff903c22edd068d97f/stdlib/sql/to.go#L193-L205

Ultimately, any table returned by a table stream function will create a table builder.  Normally it would happen right in the `Process()` method, but `sql.to` is a little more convoluted than other functions, so it happens in a function called from there instead:
https://github.com/influxdata/flux/blob/cb41c28d3391d1fc0285e3ff903c22edd068d97f/stdlib/sql/to.go#L317-L322

In this case I see that `AddTableCols` is immediately called, meaning that the output table will have exactly the same columns as the input table.  This being the case, the shape of the input and output data for `sql.to` will be the same.